android/app/build.gradle

```diff
@@ -47,10 +50,24 @@ android {
         applicationId "com.vladimir.hodaryonok.star_movie"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 25
+        targetSdkVersion 33
```

Comment:
-//-

minSdkVersion 25 is too high. let's keep at least 21.

if u will have some free time - u can read [this](https://developer.android.com/about/versions) doc and try to remember/note somewhere 2-3 new features from lollipop to android 13, that can be useful for u